File: erp-app/src/app/modules/setup/models/client-vendor.dto.ts

export class ClientVendorDTO {
    id: number;
    code: string;
    isActive: boolean;
    fullName: string;
    address: string;
    imageUrl: string;
    phoneNumber1: string;
    phoneNumber2: string;
    debit: number//سعر 
    credit: number//سعر 
    notes: string;
    idNumber: string;
    imageBase64: string;
    typeId: ClientVendorTypeEnum;
    companyId: number;
    constructor() {
        this.isActive = true;
    }

}


export enum ClientVendorTypeEnum {
    Client = 1,
    Vendor = 2,
    All = 3,
}

File: ERPServer/Shared/Shared/Entities/Setup/ClientVendorDTO.cs
using Shared.Entities.Shared;
using Shared.Enums;
using System;
using System.Collections.Generic;
using System.Text;

namespace Shared.Entities.Setup
{
    public class ClientVendorDTO:BaseDTO
    {
        public bool IsActive { get; set; }
        public string Code { get; set; }
        public string FullName { get; set; }
        public string Address { get; set; }
        public string PhoneNumber1 { get; set; }
        public string PhoneNumber2 { get; set; }
        public string ImageUrl { get; set; }
        public decimal Debit { get; set; }
        public decimal Credit { get; set; }
        public decimal OppeningBalance { get; set; }
        public string Notes { get; set; }
        public string IdNumber { get; set; }
        public string ImageBase64 { get; set; }
        public ClientVendorTypeEnum TypeId { get; set; }
        public long? CompanyId { get; set; }

    }
}
